plugins/pdr_deterministic_plugin/tests/simulation_telemetry.py

```diff
@@ -344,22 +370,25 @@ def check_logs(config):
         return 1
 
     saved_files_tests = True
-    locations_telemetry_csv = "/opt/ufm/ufm_plugin_pdr_deterministic/datastore"
-    OUTPUT_FILE_FORMAT = "%Y_%m_%d_%H_%M_%S.csv"
-
+    
     for option_location in ['abs','delta']:
-        input_path_dir = Path(join(locations_telemetry_csv,option_location))
+        input_path_dir = Path(join(TELEMETRY_DATASTORE_LOCATION,option_location))
         files = list(input_path_dir.glob("*.csv"))
-        
+
         if len(files) == 0: # testing we have files there
-            print(f"There is no files in the datastore location:{join(locations_telemetry_csv,option_location)}")
+            print(f"There is no files in the datastore location:{join(TELEMETRY_DATASTORE_LOCATION,option_location)}")
             saved_files_tests = False
             continue
+        print_test_result("there are only 10 items in the set, The test copy in each iterations more logs.",
```

Comment:
I don't understand the log message.
Can you please rewrite it to be clearer?

Reply:
edited